engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3850,6 +3897,7 @@ private Option add(Option opt)
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
     final Option force = add(new Option("force", "Force launch the application. Do not check for compatibility"));
+    final Option onlyConfigApps = add(new Option("onlyConfigApps","Only use the apps present in the config file."));
```

Comment:
"config package file". We should make it very clear that it's different from the -conf parameter, which takes a config xml file.